autogpt_platform/backend/backend/blocks/basic.py

```diff
@@ -19,14 +19,12 @@ class StoreValueBlock(Block):
     """
 
     class Input(BlockSchema):
-        input: Any = Field(
-            description="Trigger the block to produce the output. "
-            "The value is only used when `data` is None."
+        trigger: Any = SchemaField(
+            description="Trigger the block to produce the output. Passed value is never used."
         )
-        data: Any = Field(
+        value: Any = SchemaField(
             description="The constant data to be retained in the block. "
             "This value is passed as `output`.",
-            default=None,
```

Comment:
This will end up requiring the two input pins to be defined. We should make it only one input pin instead.

I see this used in other places in the code, so you need to change it to make the test work.